code/modules/modular_computers/hardware/card_slot.dm

```diff
@@ -116,7 +116,8 @@
 	usage_flags = PROGRAM_PDA
 
 /obj/item/stock_parts/computer/card_slot/Destroy()
-	loc.verbs -= /obj/item/stock_parts/computer/card_slot/proc/verb_eject_id
+	if(loc)
+		loc.verbs -= /obj/item/stock_parts/computer/card_slot/proc/verb_eject_id
```

Comment:
Have to say, this makes me a wonder if there's a possibility for verbs to remain added to objects if the card slot is ever removed via non-deconstruction means

Reply:
Sort of unclear; this part is PDA-only, I believe, and not really removable. Could use closer examination, though.